mssql-native/src/main/java/org/ballerinalang/mssql/nativeimpl/CallProcessor.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.mssql.nativeimpl;
+
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BObject;
+import org.ballerinalang.mssql.parameterprocessor.MssqlResultParameterProcessor;
+import org.ballerinalang.mssql.parameterprocessor.MssqlStatementParameterProcessor;
+
+/**
+ * This class holds the utility methods involved with executing the call statements.
+ *
+ * @since 1.2.0
+ */
+public class CallProcessor { 
```

Comment:
Since this contains utility functions, shall we rename this to `CallProcessorUtils`

Reply:
Fixed with [`ffca7b8`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/ffca7b8c8fd300371d12e74c6379c15f19c210ad)